docs/source/blog/derotation.md

```diff
@@ -0,0 +1,117 @@
+:::
+
+### 2. High-Level Pipelines
+For an end-to-end workflow, `derotation` provides two pre-built pipeline classes that automate the entire process from raw data to a corrected movie file.
+
+* **`FullPipeline`**: Designed for experiments with randomized rotations. It automatically parses analog signals, interpolates angles, uses Bayesian optimization to find the center of rotation, and performs the derotation.
+* **`IncrementalPipeline`**: Tailored for continuous, slow rotations.
+
+Both pipelines generate a corrected TIFF stack, metadata files, and debugging plots.
+
+---
+
+## Finding the Center of Rotation
+
+Accurately identifying the **center of rotation** is critical. An incorrect center will result in residual motion, where stationary objects appear to trace circles in the final movie.
+
+![](https://raw.githubusercontent.com/neuroinformatics-unit/derotation/main/docs/source/_static/wrong_center.png)
+> When the center of rotation is miscalculated, a stationary cell's center (red crosses) will appear to move in a circular path across different rotation angles.
+
+The package implements two methods to find this center:
+
+1.  **Bayesian Optimization**: Used by the `FullPipeline`, this method is computationally intensive but robust. It iteratively tests potential center coordinates to find the one that minimizes motion in the corrected movie.
+2.  **Ellipse Fitting**: The `IncrementalPipeline` tracks the path of a bright, stable object (like a cell) across frames. In a physical rotating system, this path would likely form an ellipse. The center of the fitted ellipse is an accurate estimate of the true center of rotation.
+
+---
+
+## Verification and Synthetic Data
+
+To ensure the quality of the correction and validate the algorithms, `derotation` includes tools for generating and processing synthetic data.
+
+* **`Rotator` Class**: This class can take a static image and apply a virtual line-by-line rotation, simulating the distortion from a real microscope.
+* **`SyntheticData` Class**: This builds on `Rotator` to create complete, simulated datasets, including fake cell images and corresponding rotation signals.
+
+![](https://raw.githubusercontent.com/neuroinformatics-unit/derotation/main/docs/source/_static/rotator.gif)
+> An animation showing a synthetic dataset of two "cells" being rotated with line-scanning artifacts.
+
+These tools are invaluable for testing and can even simulate complex cases like **out-of-plane rotations**, where the axis of rotation is tilted relative to the imaging plane. In such cases, the package can fit an ellipse to the object's trajectory to calculate the 3D rotation geometry and apply a more sophisticated correction.
+
+---
+
+## Get Involved
+
+`derotation` is an open-source project in active development. Community contributions are welcome.
+
+* **Report Bugs or Request Features**: [Open an issue on GitHub](https://github.com/neuroinformatics-unit/derotation/issues)
```

Comment:
Maybe the things in bold do not need to have all words capitalised?

Reply:
Agree. We should, as far as possible, to try to stick to a consistent style across the website too.